plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -217,6 +223,17 @@ public void testShowColumns()
         assertEquals(actual.getMaterializedRows().get(8).getField(1), "varchar(79)");
     }
 
+    @Test
+    @Override
+    public void testSortItemsReflectedInExplain()
+    {
+        // The format of the string representation of what gets shown in the table scan is connector-specific
+        // and there's no requirement that the conform to a specific shape or contain certain keywords.
+        assertExplain(
```

Comment:
This should work just by setting `SUPPORTS_TOPN_PUSHDOWN` to false in `hasBehaviour` instead of override.

Reply:
Indeed. Thank you for the information @hashhar .